scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -388,14 +388,14 @@ def build_reborn_binary() -> Path:
             features,
             "--bin",
-            "ironclaw-reborn",
+            "ironclaw",
         ],
         cwd=ROOT,
         env=build_env,
     )
     binary = _reborn_binary()
     if not binary.exists():
-        raise LiveQaError(f"ironclaw-reborn binary was not produced at {binary}")
+        raise LiveQaError(f"ironclaw binary was not produced at {binary}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)